README.md

```diff
@@ -22,11 +22,11 @@ Over 3200 Free SVG icons for popular brands. See them all on one page at <a href
 
 ### General Usage
 
-Icons can be downloaded as SVGs directly from <https://simpleicons.org/> - simply click the download button of the icon you want, and the download will start automatically.
+Icons can be downloaded as SVGs directly from [simpleicons.org](https://simpleicons.org) - simply click the download button of the icon you want, and the download will start automatically.
```

Comment:
The link is repeated, we can use a [reference link](https://spec.commonmark.org/0.31.2/#reference-link).

Reply:
There are three texts that use the `https://simpleicons.org`:

- `<a href="https://simpleicons.org">SimpleIcons.org</a>` - This one cannot use markdown since it's in an HTML tag with newline
- `<https://simpleicons.org/>`
- `[simpleicons.org website](https://simpleicons.org/)`

It's OK not to use a reference link since we don't have too much repeated content for this link.

They also have different display texts.